ingest/router.go

```diff
@@ -195,7 +196,7 @@ type BatchPayload struct {
 func (r *Router) sendToRotor(c *gin.Context, ingestMessageBytes []byte, stream *StreamWithDestinations, sendResponse bool) (asyncDestinations []string, tagsDestinations []string, rError *appbase.RouterError) {
 	var err error
 	if stream.BackupEnabled {
```

Comment:
As an alternative, is it possible to disable this backup stream? We would be OK with that approach.

if there is an env var for this config, great! If not, I can use this PR to implement it instead.

Reply:
@rodfsoares 
backup can be disabled on Workspace level by setting `featuresEnabled` column of **Workspace** table to `{nobackup}` value.
Also backup topics should not be enabled by default - that is a bug.
We are going to fix that.

Also changing the topic name like that will break things on setups where backup topics are consumed.

Reply:
> backup can be disabled on Workspace level by setting featuresEnabled column of Workspace table to {nobackup} value.

@absorbb, got it, I'll connect to the DB (e.g. psql) and set that value.

once I confirm it's working, I'll close this PR, thanks 🙏